src/services/leafletLayerHelpers.js

```diff
@@ -55,6 +55,14 @@ angular.module('ui-leaflet')
                 return L.tileLayer(url, params.options);
             }
         },
```

Comment:
ui-leaflet not provide a way for add more type layer from a decorator, it's a local function into layer helpers.